src/pages/stake/ValidatorActions.tsx

```diff
@@ -31,6 +32,8 @@ const ValidatorActions = ({ destination }: { destination: ValAddress }) => {
   const { data: rewards, ...rewardsState } = useRewards()
   const state = combineState(delegationState, delegationsState)
   const calcValue = useMemoizedCalcValue()
+  const readNativeDenom = useNativeDenoms()
+  const { decimals } = readNativeDenom(chain?.baseAsset ?? "")
```

Comment:
I feel like all the processes to get `decimals` and `token` from the `readNativeDenom` function should be done in one line.  Relevant arguments should be passed into a function and the relevant characteristics returned.  Just so we have a central location where these processes are taking place and it will be easier to extract the values.

Reply:
yea this is what I was saying during standup yesterday. everything should be handled long-term in the `read` component but this is a temp fix for now.